ang/civicase/case/details/people-tab/services/people-tab-roles.service.js

```diff
@@ -24,12 +24,19 @@
     roles.updateRolesList = updateRolesList;
 
     /**
-     * Assign number of roles present per type of relationship
+     * Assign number of roles present per type of relationship.
+     *
+     * For case roles, it will only count them if they are active.
      */
     function assignCountOfRolesPerType () {
       _.each(roles.caseTypeRoles, function (caseTypeRole) {
         caseTypeRole.count = _.filter(roles.list, function (role) {
-          return role.display_name && role.name === caseTypeRole.name;
+          var roleIsAssigned = !!role.display_name;
+          var roleBelongsToType = role.role === caseTypeRole.role;
+          var isClientRole = role.role === ts('Client');
+
+          return roleIsAssigned && roleBelongsToType &&
```

Comment:
In the PR doc it said 
> Inactive case roles would be taken into consideration

But how is that fixed here? Is `!role.isPastRole` related?

Reply:
@deb1990 they are synonymous:

https://github.com/compucorp/uk.co.compucorp.civicase/blob/e1d0083624990a57a1296b9be6e536beccf8f36e/ang/civicase/case/details/people-tab/services/people-tab-roles.service.js#L140

The `isPastRole` property is set to true if the relationship associated to that particular role is not active.

Reply:
Thanks. Now I know this part was not changed in this PR, but to be honest its a bit confusion.
If we used the `isActive` naming convention then its easier to understand. 
The question I had, someone else can have it too, so having the same name would make it easier to understand/relate.
So if its not going to take a lot of time, can we change this logic to use `isActive` naming convention?